pkgs/development/tools/rust/cargo-careful/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage {
+  pname = "cargo-careful";
+  version = "master";
```

Comment:
See comment below for the relevant documentation link.

Reply:
I think version pin also resolves this.